src/components-examples/angular/textarea/textarea-forms/textarea-forms-example.css

```diff
@@ -0,0 +1,3 @@
+.sbb-textarea {
+  min-height: 6rem !important;
```

Comment:
Is the important now really necessary?

Reply:
seems do depend whether we test no @arciela s computer or mine. Will investigate further

Reply:
not needed anymore